readthedocs-theme/templates/readthedocs/product.html

```diff
@@ -0,0 +1,256 @@
+{% extends "page.html" %}
+
+{% block body_id %}product{% endblock %}
+
+{% block content %}
+
+  <!-- How it works -->
+  <section class="ui vertical segment very padded" id="how-it-works">
+    <div class="ui container center aligned">
```

Comment:
If we don't have anything here, it's probably best to remove this section and come back to it later. We want to be moving towards less mockup copy and content in the site.

Reply:
Removed this section.